### PR TITLE
qcore: add a bosic tox.ini

### PR DESCRIPTION
**Pull request by SurenNihalani**
for quora's internal deployment system integration

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist =
+    py27,py36
+
+[testenv]
+deps =
+    -rrequirements.txt
```
Comment (JelleZijlstra) [inline tox.ini]: missing a space?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist =
+    py27,py36
```
Comment (JelleZijlstra) [inline tox.ini]: let's also test 3.4 and 3.5 and allow missing interpreters (similar to pyanalyze's tox.ini)

Comment (SurenNihalani) [reply tox.ini]: Why do you want to allow missing interpreters?

File: tox.ini (hunk 1 above)
Comment (SurenNihalani) [reply tox.ini]: This is actually correct

File: tox.ini (hunk 2 above)
Comment (SurenNihalani) [reply tox.ini]: okay. I misunderstood the value of this option. added.

**Comment by SurenNihalani** [comment]
gonna merge this to unblock me. will add amending commits if needed.

**Pull request merged**